src/Stratis.Bitcoin.IntegrationTests.Common/EnvironmentMockUpHelpers/CoreNode.cs

```diff
@@ -214,11 +230,39 @@ private void StartStratisRunner()
                 cancellationToken: new CancellationTokenSource(timeToNodeStart).Token,
                 failureReason: $"Failed to achieve state = started within {timeToNodeStart}");
 
-            if (this.builderNotInIbd)
+            if (this.builderNotInIBD)
                 ((InitialBlockDownloadStateMock)this.FullNode.NodeService<IInitialBlockDownloadState>()).SetIsInitialBlockDownload(false, DateTime.UtcNow.AddMinutes(5));
 
             if (this.builderWithWallet)
                 this.Mnemonic = this.FullNode.WalletManager().CreateWallet(this.builderWalletPassword, this.builderWalletName, this.builderWalletPassphrase);
+
+            if (this.builderNoValidation)
+                DisableValidation();
+        }
+
+        public void DisableValidation()
```

Comment:
OK yeah I have removed this switch 👍 and will just pass in a network that has no validation.

Reply:
Actually, this isn't going to work as we always construct the rule set when we start up the node and then set the rules on Consensus after the fact. So in the tests we first need to start up the node and THEN clear the rules.

Reply:
We can however pass in another network instance to this node that requires no validation, so that we don't accidentally affect the node that DO require validation ;)

Reply:
yes that's what I mean either pass an instance or a generic instance and create the network and remove the rules.